ivy_tests/test_ivy/test_frontends/test_numpy/test_statistics/test_order_statistics.py

```diff
@@ -6,10 +6,42 @@
 from ivy_tests.test_ivy.test_functional.test_core.test_statistical import (
     _statistical_dtype_values,
 )
+import ivy_tests.test_ivy.helpers as helpers
 import ivy_tests.test_ivy.test_frontends.test_numpy.helpers as np_frontend_helpers
 from ivy_tests.test_ivy.helpers import handle_frontend_test
 
 
+# ptp
+@handle_frontend_test(
+    fn_tree="numpy.ptp",
+    dtype_values_axis=_statistical_dtype_values(function="ptp"),
+    keep_dims=st.booleans(),
+)
+def test_numpy_ptp(
+    dtype_values_axis,
+    where,
+    frontend,
+    test_flags,
+    fn_tree,
+    on_device,
+    keep_dims,
+):
+    input_dtypes, values, axis = dtype_values_axis
+    if isinstance(axis, tuple):
+        axis = axis[0]
+
+    helpers.test_frontend_function(
+        a=values,

```

Comment:
As generated `values` is a list containing one array you will unpack it
```suggestion
        a=values[0],

```